src/main/java/org/kohsuke/groovy/sandbox/StackVariableSet.java

```diff
@@ -14,13 +14,17 @@
  *
  * @author Kohsuke Kawaguchi
  */
-final class StackVariableSet {
+final class StackVariableSet implements AutoCloseable {
```

Comment:
Small redesign to allow this to be used in `try`-with-resources blocks rather than a closure.